common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -238,8 +256,61 @@ export class SendTransaction extends React.Component {
     return { to, data, value, unit, gasLimit, readOnly };
   }
 
+  isValid() {
+    return (
+      isValidETHAddress(this.state.to) &&
+      this.state.value &&
+      !isNaN(Number(this.state.value)) &&
+      isFinite(Number(this.state.value))
+    );
+  }
+
+  // FIXME MOVE ME
+  getTransactionFromState() {
+    // FIXME add gas price
+    if (this.state.unit === 'ether') {
+      return {
+        to: this.state.to,
+        from: this.props.wallet.getAddress(),
+        // gasPrice: `0x${new Number(50 * 1000000000).toString(16)}`,
+        value: '0x' + toHex(toWei(Big(this.state.value), 'ether'))
+      };
+    }
+    const token = this.props.tokens.find(x => x.symbol === this.state.unit);
+    if (!token) {
+      return;
+    }
+
+    return {
+      to: token.address,
+      from: this.props.wallet.getAddress(),
+      // gasPrice: `0x${new Number(50 * 1000000000).toString(16)}`,
+      value: '0x0',
+      data: `0xa9059cbb${padLeft(stripHex(this.state.to), 64)}${padLeft(
```

Comment:
It is a part of erc20 `transfer` function signature hash. When you want to call contract in eth you hash function name+argument types and pass part of it with tx.